Classes/ConfigurationTest.php

```diff
@@ -42,14 +39,14 @@ public function testCommand($params)
                 continue;
             }
 
-            $header = sprintf('%s%s', strtoupper($fileExtension), $fileIsUploaded ? ' on Upload' : '');
-            $file = sprintf(
+            $header = \sprintf('%s%s', \strtoupper($fileExtension), $fileIsUploaded ? ' on Upload' : '');
```

Comment:
Please remove the backslashes. I'm aware of this micro performance optimization, but it's nothing compared to spin up a new process and optimize an image.